tests/sanity/test/explorer/test-api-datasets-v2.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2018, 2019
+ */
+
+const expect = require('chai').expect;
+const utils = require('./utils');
+
+let REQ;
+const TEST_DATASET_PATTERN = 'SYS1.LINKLIB*';
+const TEST_DATASET_NAME = 'SYS1.LINKLIB';
+const TEST_DATASET_MEMBER_NAME = 'ACCOUNT';
+
+describe('test explorer server datasets api', function() {
+  before('verify environment variables', function() {
+    // allow self signed certs
+    process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+    // sets up a request variable and retrieves the authentication needed for v2 APIs
+    REQ = utils.verifyAndSetupEnvironment();
+  });
+
+  it(`should be able to list data sets of ${TEST_DATASET_PATTERN} (v2 API)`, async() => {
+    const uuid = utils.uuid();
+    const authenticationCookie = await utils.login(uuid);
+
+    utils.log(uuid, ' URL: /api/v2/datasets/' + encodeURIComponent(TEST_DATASET_PATTERN));
+    const res = await REQ.get('/api/v2/datasets/' + encodeURIComponent(TEST_DATASET_PATTERN), {
```

Comment:
I think this test supersedes `tests/sanity/test/apiml/test-mvs-via-gateway.js`, we can remove the other one.